docs/ballerina-by-example/examples/inbound-request-with-multiparts/inbound-request-with-multiparts.bal

```diff
@@ -54,12 +56,15 @@ function writeToFile(io:ByteChannel byteChannel) {
     string dstFilePath = "./files/savedFile.ppt";
     io:ByteChannel destinationChannel = getByteChannel(dstFilePath, "w");
     blob readContent;
-    int numberOfBytesRead;
-    readContent, numberOfBytesRead = byteChannel.readAllBytes();
-    int numberOfBytesWritten = destinationChannel.writeBytes(readContent, 0);
+    int numberOfBytesRead = 1;
+    while (numberOfBytesRead != 0) {
+        readContent,numberOfBytesRead = byteChannel.readBytes(10000);
+        int numberOfBytesWritten = destinationChannel.writeBytes(readContent, 0);
+    }
 }
 
 function getByteChannel (string filePath, string permission) (io:ByteChannel) {
-    io:ByteChannel channel = io:openFile(filePath, permission);
+    file:File src = {path:filePath};
+    io:ByteChannel channel = src.openChannel(permission);
```

Comment:
Why was the previous way of getting a channel changed? The `openChannel()` function in `ballerina.file` will be removed.